### ISSUE TITLE
Icon generator does not work and SSL not recognized

### ISSUE THREAD

**Issue by whiwo** [pwa-builder/pwabuilder-blog#22]
I was going from scratch to use your builder but got immediately stuck after uploading a 512x512 icon. It hangs for several minutes and then goes back to the default upload. I tried to generate screenshots but that doesn't work either.

Since I had already created my icons, I followed the instructions and zipped them up to the folder 'images' and uploaded them to my root along with the previous manifest I had created and the new code for background sync. Once tested, everything was working as it should except SSL. 

SSL is the one thing that always worked when it checked for encryption but for some reason it no longer is. 

In short, it does kinda work though I haven't tested it in Lighthouse yet but the SSL issue needs to be resolved. I have it on Cloudfare who handles my DNS records so there's no reason to have it on my host (and it won't allow me to install it there anyway because of this).

**Comment by whiwo** [comment]
I switched to Chrome and now the SSL is working. However, it would be nice if the icon generator and screenshots were being generated so that I know my sizes are correct. Anyway, all else appears to be good. https://partyfavorz.com/

**Comment by whiwo** [comment]
The icon and screenshot generator are now working. The main issue here is that I cannot download the images to upload them into my root. The images I currently have there were generated elsewhere. I tried downloading all the files thinking the images may be included but they're not. So, that leaves the download option underneath the icons and that does not work.

After reading the instructions, it states that all the icons must be in a zip folder loaded into the root (which is new) but I need the correct icons for it to reference them and I can't do that if I can't download them. 

Lighthouse is recognizing my PWA and it is installable however to issues persist in that the 512x512 square icon does not exist when it does (it was the base for generating all the other icons) and is not maskable (when it is png). 

There appears to be an issue with the lazy load function of my site but it's use is imperative. My theme has continuous scroll so that function must continue to be enabled.  See the message from Bing below.

See attachments
![Lighthouse-new](https://user-images.githubusercontent.com/7907882/123806917-66337900-d8bd-11eb-9769-37d81356c39c.PNG)
![Manifest](https://user-images.githubusercontent.com/7907882/123806934-69c70000-d8bd-11eb-9102-f14cc57ab62c.PNG)

**Comment by JudahGabriel** [comment]
Hi @whiwo 

You've opened an issue on our Blog repo. Can you instead [open an issue on our main repo](https://github.com/pwa-builder/pwabuilder/issues)?
